UserList/UserList/Controller/HomeViewController.swift

```diff
@@ -17,9 +18,14 @@ final class HomeViewController: UIViewController {
     private var tapIndex: Int?
     private var searchUsers = [User]()
     private var searching = false
+    private var firstLoad = true
```

Comment:
Sai convention của boolean. isSearching và isFirstTimeLoading nhé. Thiếu tiền tố is